### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by michal-raska**

### CODE REVIEW

**Review by jakubhava — COMMENTED** [review]
The download button and build passing button are still pointing to the original repo, otherwise good

**Comment by michal-raska** [comment]
@jakubhava Thanks, updated. I'll wait for the maven central to update and I'll merge.

File: README.md [hunk 1]
```diff
@@ -1,8 +1,6 @@
 xgboost-predictor-java
 ======================
-
-[![Build Status](https://travis-ci.org/komiya-atsushi/xgboost-predictor-java.svg?branch=master)](https://travis-ci.org/komiya-atsushi/xgboost-predictor-java)
-[![Download](https://api.bintray.com/packages/komiya-atsushi/maven/xgboost-predictor/images/download.svg) ](https://bintray.com/komiya-atsushi/maven/xgboost-predictor/_latestVersion)
+[![Maven Central](https://maven-badges.herokuapp.com/maven-central/ai.h2o/xgboost-parser/badge.svg)](https://maven-badges.herokuapp.com/maven-central/ai.h2o/xgboost-parser)
```
Comment (jakubhava) [inline README.md]: almost good, just poiting to xgboost-parser instaed of xgboost-predictor :)

Comment (michal-raska) [reply README.md]: omg a stupid mistake, thanks for double check!

**Pull request merged**